Yosemite/YosemiteTests/Stores/AppSettingsStoreTests.swift

```diff
@@ -519,6 +519,127 @@ final class AppSettingsStoreTests: XCTestCase {
         XCTAssertTrue(isEnabled)
     }
 
+    func test_loadJetpackBenefitsBannerVisibility_returns_true_on_new_generalAppSettings() throws {
+        // Given
+        try fileStorage?.deleteFile(at: expectedGeneralAppSettingsFileURL)
+
+        let currentTime = Date(timeIntervalSince1970: 1638105924)
```

Comment:
Minor nit: It would be great to add a comment explaining that just any random time here should work. I was a bit confused at first about when this time is 😅

Reply:
Good point! Added some comments in https://github.com/woocommerce/woocommerce-ios/pull/5505/commits/5d82aedc78b7120b7d8d1845750712195832e9d7